core-bundle/src/String/HtmlAttributes.php

```diff
@@ -419,34 +418,30 @@ private function escapeValue(string $name, string $value): string
     private function parseStyles(string $styles): array
     {
         // Regular expression to match declarations according to https://www.w3.org/TR/css-syntax-3/#declaration-list-diagram
-        $declarationRegex = <<<'EOD'
-            (
-                (?:
-                    \\.                           # Escape
-                    |"(?:\\.|[^"\n])*+(?:"|\n|$)  # String token double quotes
-                    |'(?:\\.|[^'\n])*+(?:'|\n|$)  # String token single quotes
-                    |\{(?:(?R)|[^}])*+(?:\}|$)    # {}-block
-                    |\[(?:(?R)|[^]])*+(?:\]|$)    # []-block
-                    |\((?:(?R)|[^\)])*+(?:\)|$)   # ()-block
-                    |[^;{}\[\]()"']               # Anything else
-                )++
-            )ixs
-            EOD;
+        $declarationRegex = '(
+            (?:
+                \.                              # Escape
+                |"(?:\.|[^"\n])*+(?:"|\n|$)     # String token double quotes
+                |\'(?:\.|[^\'\n])*+(?:\'|\n|$)  # String token single quotes
+                |\{(?:(?R)|[^}])*+(?:}|$)       # {}-block
+                |\[(?:(?R)|[^]])*+(?:]|$)       # []-block
+                |\((?:(?R)|[^\)])*+(?:\)|$)     # ()-block
+                |[^;{}\[\]()"\']                # Anything else
+            )++
+        )ixs';
 
         // Regular expression to match an <ident-token> according to https://www.w3.org/TR/css-syntax-3/#ident-token-diagram
-        $propertyRegex = <<<'EOD'
-            (
-                ^
-                (?!\d)                             # Must not start with a digit
-                (?!-\d)                            # Must not start with a dash followed by a digit
-                -?+                                # Optional leading dash
-                (?:
-                    [a-z0-9\x80-\xFF_-]
-                    |\\(?:[0-9a-f]{1,6}\s?|[^\n])  # Escape
-                )++
-                $
-            )ixs
-            EOD;
+        $propertyRegex = '(
+            ^
+            (?!\d)                               # Must not start with a digit
+            (?!-\d)                              # Must not start with a dash followed by a digit
+            -?+                                  # Optional leading dash
+            (?:
+                [a-z0-9\x80-\xFF_-]
+                |\\\\(?:[0-9a-f]{1,6}\s?|[^\n])  # Escape
```

Comment:
@ausi I had to convert `\\` into `\\\\` here to get rid of the `preg_match(): Unknown modifier '+'`. Is the change correct?

Reply:
```suggestion
                |\\\(?:[0-9a-f]{1,6}\s?|[^\n])  # Escape
```
The change is correct, but as our coding style is to only escape backslashes if needed, `\\\` would be the right one I think.